README.md

```diff
@@ -0,0 +1,378 @@
+        * isoweb, isostore, isolocal or isoclient: the source of the iso file used to build the OS image. At least one of those four should be specified. If more then one is specified, on-static will use isostore over isolocal over isoweb over isoclient. 
+            * isoweb: use iso file from web, can be http or ftp, like _http://example.com/example.iso_. **Https** is not tested yet.
+            * isolocal: use iso file from the server which on-static in running.
+            * isoclient: use iso file uploaded from user client where the APIs are called. Iso files are uploaded using HTTP PUT method.
+            * isostore: use in-store iso file that had been uploaded before from above three sources. This is useful when you are adding a OS image that had been removed earlier.
+
+            Using **isoclient**
+
+            ```
+            curl -X PUT "http://10.62.59.150:7070/images?name=photon&version=1.0&isoclient=client.iso" --upload-file path-to-file/test.iso
+            Uploaded 100 %
+            Upload finished!
+            ```
+
+            Using **isoweb**. image status will set as 'downloading iso' and iso download will be carried out at the background. You can check the status afterwards using get/list image API. 
+
+            ```
+            curl -X PUT "http://10.62.59.150:7070/images?name=photon&version=1.0&isoweb=http://10.62.59.150:9090/iso/photon-1.0.iso"
+            {
+                "id": "39647624-e640-41d0-901b-afc58af98725",
+                "iso": "photon-1.0.iso",
+                "name": "photon",
+                "version": "1.0",
+                "status": "downloading iso"
+            }
+            ```
+
+            Using **isolocal**.
+
+            ```
+            curl -X PUT "http://10.62.59.150:7070/images?name=centos&version=7.0&isolocal=/home/onrack/github/on-static/static/files/iso/centos-7.0.iso"
+            {
+                "id": "9fce7e8f-c7ef-49db-a47f-1924675d5e29",
+                "iso": "/home/onrack/github/on-static/static/files/iso/centos-7.0.iso",
+                "name": "centos",
+                "version": "7.0",
+                "status": "preparing"
+            }
+            ```
+
+            Using **isostore**.
+
+            ```
+            curl -X PUT "http://10.62.59.150:7070/images?name=centos&version=7.0&isostore=centos-7.0.iso"
+            {
+                "id": "b6b3e3be-c799-4af4-86c8-09a99d3aa7c7",
+                "iso": "centos-7.0.iso",
+                "name": "centos",
+                "version": "7.0",
+                "status": "preparing"
+            }
+            ```
+
+    3. DELETE http://0.0.0.0:7070/images: delete an OS images. two parameters are needed.
+        * name: in query or body, the OS name.
+        * version: in query of body, the OS version.
+
+        ```
+        curl -X DELETE -H "Content-Type: application/json" -d '' "http://10.62.59.150:7070/images?name=centos&version=7.0"
```

Comment:
so there's no API to delete an /images/<id> API?   by the way, one experience of RackHD is response is [] after calling API to delete one entitiy